templates/base/packages/nextjs/components/ScaffoldEthAppWithProviders.tsx.template.mjs

```diff
@@ -4,7 +4,7 @@ const contents = ({ providerNames, providerSetups, providerImports, providerProp
   // filter out empty strings
   const providerOpeningTags = providerNames.filter(Boolean).map((name, index) => `<${name} ${providerProps[index]}>`);
 
-  const providerClosingTags = providerNames.filter(Boolean).map(name => `</${name}>`);
+  const providerClosingTags = providerNames.filter(Boolean).map(name => `</${name}>`).reverse();
```

Comment:
Didn't get this why are we reversing this?

Reply:
because if we have multiple providers we want them to be
```
<ProviderOne>
    <ProviderTwo>
    ...
    </ProviderTwo>
</ProviderOne>
```
but not
```
<ProviderOne>
    <ProviderTwo>
    ...
    </ProviderOne>
</ProviderTwo>
```

Reply:
Ohh shit yes! It previously worked in case `-e subgraph` because we only had one provider.